calculator/operations.py

```diff
@@ -1,23 +1,43 @@
+from __future__ import division
 
 def add(*args):
-    # your implementation here
-    pass
+        if len(args) == 1:
```

Comment:
Of course the use of len is to avoid exceptions in the sum function. Use lambda in what? Reduce function? I avoid it for multiple reasons. Style and speed basically